pytorch_lightning/core/datamodule.py

```diff
@@ -247,22 +251,6 @@ def prepare_data(self, *args, **kwargs):
     def setup(self, stage: Optional[str] = None):
         pass
 
-    @abstractmethod
```

Comment:
this just solves an IDE issue, nothing much, since some are not really abstract methods. This is already inherited from DataHooks. So not a significant change.